qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/basic_fitting_context.py

```diff
@@ -37,6 +37,10 @@ def __init__(self, allow_double_pulse_fitting: bool = False):
         self._chi_squared_for_undo: list = []
 
         self._plot_guess: bool = False
+        self._plot_guess_type: str = None
```

Comment:
Should this have a default value? For instance we might want it defaulted to `PLOT_GUESS_FIT_RANGE` (see comment below)